### PR TITLE
refactor: Use gin to replace mux

### PR DESCRIPTION
**Pull request by howjmay**
Travis CI s been added as CI tool from this PR.

### CODE REVIEW

**Comment by jserv** [comment]
Since you are making non-trivial changes, there should be a solid and self-contained way to validate the change. Can you follow [golang-gin-realworld-example-app](https://github.com/gothinkster/golang-gin-realworld-example-app) and provide necessary tests?

**Comment by howjmay** [comment]
> Since you are making non-trivial changes, there should be a solid and self-contained way to validate the change. Can you follow [golang-gin-realworld-example-app](https://github.com/gothinkster/golang-gin-realworld-example-app) and provide necessary tests?

In this PR, I have provided a basic test for `/ping` to ensure the ability.

**Comment by howjmay** [comment]
@junwei0117 Had implemented a [demokit](https://github.com/DropKit/demokits) which can be used as a comprehensive test for the routing as well.

**Comment by jserv** [comment]
> In this PR, I have provided a basic test for `/ping` to ensure the ability.

Good. Let's enable CI before merging this pull request.

**Comment by HowJMay** [comment]
> > In this PR, I have provided a basic test for `/ping` to ensure the ability.
> 
> Good. Let's enable CI before merging this pull request.

I am trying to translate and simplify the demokit to python script, but it will still take around 200 lines of python code. Could I split the testing script to another PR?

File: tests/index.ts [hunk 1]
```diff
@@ -0,0 +1,253 @@
+import axios from "axios";
+import colors from "colors";
+import {
+  sleep
+} from "sleep";
+
+const SHOW_RESPONSE = true
+
+const DROPKIT_ENDPOINT = "http://localhost:5000";
```
Comment (jserv) [inline tests/index.ts]: Read configurations for test suite from specific file(s).

**Review by jserv — CHANGES_REQUESTED** [review]
Read test programs in Go as well.

**Pull request merged**